### PR TITLE
Spear is now craftable

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

Makes the spear craftable via the craft menu instead of wonky bay material code
Icon ingame
image
examination
image
in right hand wielded
image
on back
image
info tab
image
crafting menu
image

## Why It's Good For The Game

Consistency in makeshift weapons, fuck bay material code, we all love spears

## Changelog
:cl:
add: Added craftable spear via the craft menu. Sprite was modified by me but originally made by CeUvI
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
Just curious, why not a slightly stronger spear with a slightly better material than glass? This seems pretty poor as a melee weapon to be frank.

**Comment by Wouju** [comment]
> Just curious, why not a slightly stronger spear with a slightly better material than glass? This seems pretty poor as a melee weapon to be frank.

it's not meant to be a good melee weapon because it's literally a glass shard attached to 2 rods welded together

**Comment by Sparrowbird611** [comment]
> > Just curious, why not a slightly stronger spear with a slightly better material than glass? This seems pretty poor as a melee weapon to be frank.
> 
> it's not meant to be a good melee weapon because it's literally a glass shard attached to 2 rods welded together
> 
> and because it's a makeshift melee weapon that you should be able to craft easily

Which is why I recommended a more expensjve spear tip and better damage stats. Spears don't even fit in backpacks, so we might as well make them a bit more expensive and a bit more effective instead of just leaving them out to dry. If someone really wants a shitty melee blade they'll make a chopper or just grab a toolbox. The homewrecker has better stats than this and is nearly as easy to craft as this thing, not to mention fits in backpacks.

File: code/game/objects/items/weapons/tools/knives.dm [hunk 1]
```diff
@@ -293,3 +293,29 @@
...
+	armor_penetration = ARMOR_PEN_MODERATE //15
+	max_upgrades = 3
+	tool_qualities = list(QUALITY_CUTTING = 10,  QUALITY_WIRE_CUTTING = 5, QUALITY_SCREW_DRIVING = 1)
+	matter = list(MATERIAL_STEEL = 1, MATERIAL_GLASS = 1)
+	attack_verb = list("slashed", "stabbed") //there's not much you can do with a spear aside from stabbing and slashing with it
+	slot_flags = SLOT_BACK
+	structure_damage_factor = STRUCTURE_DAMAGE_BLADE
+
+	rarity_value = 20
+	spawn_tags = SPAWN_TAG_KNIFE
```
Comment (TheShown911) [inline code/game/objects/items/weapons/tools/knives.dm:321]: No newline.

**Review by TheShown911 — APPROVED** [review]

File: code/game/objects/items/weapons/tools/knives.dm (hunk 1 above)
Comment (Wouju) [reply code/game/objects/items/weapons/tools/knives.dm]: should be fine now

**Pull request merged**